CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 1.0.1 - 2022-07-26
+- migrate arriba 1.0 to Visdarr
```

Comment:
What do you think about specifying why we're tagging this as an "earlier" version? Something like 
```
- migrate arriba 1.0 to Vidarr with the same WDL that is installed on Niassa
```

Reply:
updated